v2/internal/testcommon/kube_test_context_envtest.go

```diff
@@ -54,7 +54,7 @@ import (
 )
 
 func getRoot() (string, error) {
-	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
+	cmd := exec.CommandContext(context.Background(), "git", "rev-parse", "--show-toplevel")
```

Comment:
Why make this change? I assume `Command` is calling `CommandContext` w/ background context already, so this is no different than we had before?